tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -704,6 +704,21 @@ module.exports = {
       return this
     },
 
+    cancelCopyResource: async function(resource) {
+      await this.waitForFileVisible(resource)
+
+      // Trigger move
+      await filesRow.openFileActionsMenu(resource).copy()
+
+      // cancel move
+      await this.useXpath()
+        .waitForElementVisible(client.page.filesPage().elements.cancelMoveBtn.selector)
```

Comment:
IMO, renaming this element makes more sense on what it does
```suggestion
        .waitForElementVisible(client.page.filesPage().elements.cancelMoveCopyBtn.selector)
```